### ISSUE TITLE
delete 'dbColumn' => 'image' value without deleting complete record

### ISSUE THREAD

**Issue by tonylisanti** [milesj/uploader#94]
I'm wanting to delete the files and I'm also wanting to delete the values for the Uploader.Attachment  'dbColumn' as well as the values for the transforms Uploader.Attachment  'dbColumns' with in the database. I'm not wanting to delete the complete record in the database though. I'm able to delete the files manually using   $this->Uploader->delete(); and I'm able to update the transforms values in the db to null using:

$data = array('id' => $id, 'image_original' => NULL, 'transform1' => NULL, 'transform2' => NULL);

$this->Model->save($data);

but the 'image_original' Uploader.Attachment dbColumn always retains (or is being updated again) its value(file path).

Basically i'm wanting to delete all files and all associated dbColumns (original file path and transforms) values in the db without deleting the record but no matter what I do the dbColumn associated with original image file path always retains it's value.

Any advice or help would be greatly appreciated.
Thanks,
Tony
